config/samples/datadog-operator-hub-example.yaml

```
@@ -0,0 +1,43 @@
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  clusterName: <CLUSTER_NAME>
+  credentials:
+    apiKey: <DATADOG_API_KEY>
+    appKey: <DATADOG_APP_KEY>
+    token: <DATADOG_CLUSTER_AGENT_TOKEN>
+  agent:
+    image:
+      name: "datadog/agent:7.24.1"
+    apm:
+      enabled: false
+    process:
+      enabled: true
+      processCollection: false
+    log:
+      enabled: false
+      logsConfigContainerCollectAll: false
+    systemProbe:
+      enabled: false
+      bpfDebugEnabled: false
+    security:
+      compliance:
+        enabled: false
+      runtime:
+        enabled: false
+  clusterAgent:
+    replicas: 2
+    image:
+      name: "datadog/cluster-agent:1.10.0"
```

Should we switch to the gcr images instead ?
```suggestion
      name: "gcr.io/datadoghq/cluster-agent:1.10.0"
```